src/main/java/vimification/storage/UserPrefsStorage.java

```diff
@@ -1,4 +1,4 @@
-package vimification.storage;
+package seedu.address.storage;
```

Comment:
Should it be `vimification` here? I renamed the entire package.

Reply:
I changed it back to vimification package. Should be fine now.